src/test/article.spec.js

```diff
@@ -0,0 +1,158 @@
+          expect(response.body.data.length).to.be.greaterThan(0);
+          expect(response.body.data[0]).to.have.property('id');
+          expect(response.body.data[0]).to.have.property('title');
+          expect(response.body.data[0]).to.have.property('description');
+          expect(response.body.data[0]).to.have.property('content');
+          expect(response.body.data[0]).to.have.property('image');
+          expect(response.body.data[0]).to.have.property('slug');
+          expect(response.body.data[0]).to.have.property('authorId');
+          expect(response.body.data[0]).to.have.property('status');
+          expect(response.body.data[0]).to.have.property('publisheddate');
+          expect(response.body.data[0]).to.have.property('createdAt');
+          expect(response.body.data[0]).to.have.property('updatedAt');
+          done();
+        });
+    });
+  });
+
+  describe('GET A SPECIFIC ARTICLE', () => {
+    it('should successfully get a specific', done => {
+      chai
+        .request(app)
+        .get(`${articleEndpoint}/${article.id}`)
+        .set('x-access-token', token)
+        .send()
+        .end((error, response) => {
+          expect(response.body.status).to.equal(200);
+          done();
+        });
+    });
+
+    it('should throw a 404 error when article is not found', done => {
+      chai
+        .request(app)
+        .get(`${articleEndpoint}/345`)
+        .set('x-access-token', token)
+        .send()
+        .end((error, response) => {
+          expect(response.status).to.equal(404);
+          expect(response.body.message).to.equal('Article Not Found');
+          done();
+        });
+    });
+  });
+
+  describe('UPDATE AN ARTICLES', () => {
+    before(() => {});
+
+    it('should successfully update a specific article', done => {
+      chai
+        .request(app)
+        .put(`${articleEndpoint}/${article.id}`)
+        .set('x-access-token', token)
+        .end((error, response) => {
+          expect(response.status).to.equal(200);
+          expect(response.body.message).to.equal(
+            'Article successfully updated'
+          );
+          expect(response.body.status).to.equal(200);
+          done();
+        });
+    });
+
+    it('should throw a 404 error when article is not found', done => {
+      chai
+        .request(app)
+        .put(`${articleEndpoint}/345`)
+        .set('x-access-token', token)
+        .send()
+        .end((error, response) => {
+          expect(response.status).to.equal(404);
+          expect(response.body.message).to.equal('Article Not Found');
+          done();
+        });
+    });
+  });
+
+  describe('DELETE AN ARTICLE', () => {
+    before(() => {});
+
+    it('should successfully delete an article', done => {
+      chai
+        .request(app)
+        .delete(`${articleEndpoint}/${article.id}`)
+        .set('x-access-token', token)
+        .end((error, response) => {
+          expect(response.status).to.equal(200);
```

Comment:
Should `status` be 200?

Reply:
Thank you, work is in progress to implement the feedback received on this review.
However, looking at this material: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE

I think a status code of `204` will be appropriate.